src/components/annual-report/report-data/utils/mathService.js

```diff
@@ -0,0 +1,7 @@
+export const getRoundedValue = (val, places) => {
+    if (isNaN(Number(val))) {
+        return val
+    }
+
+    return Math.round(val * Math.pow(10, places)) / Math.pow(10, places)
+}
```

Comment:
This is an elegant solution to the rounding problem 🙂 There's [`<number>.toFixed(places)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) available -- would that do what you need to do here?

Reply:
Yeah. `toFixed` would be easier. The current app rounds*, so for example 25.123 is rounded to 25.1, but 25 remains 25. `Number(25).toFixed(1)` though would result in `25.1`. I don't know if this is an important decision, so maybe it makes sense to change to avoid using our own rounding function?

* https://github.com/dhis2/who-data-quality-app/blob/master/src/appCore/appServiceMath.js#L183-L189

Reply:
Ah good point -- I think we should keep `getRoundedValue`; I think it's better and a bit more flexible